src/main/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/DraftController.java

```diff
@@ -157,4 +157,12 @@ private void assertCanEdit(Draft draft, String userId) {
             throw new AuthorizationException();
         }
     }
+
+    private int toDbId(String apiId) {
```

Comment:
Maybe document this method to explain the situation for people looking at this in the future